test/change-stream.test.js

```diff
@@ -20,14 +21,19 @@ describe('PersistedModel.createChangeStream()', function() {
       });
     });
 
+    afterEach(function() {
+      var Score = this.Score;
+      expect(Score._observers['after save']).to.be.empty();
+      expect(Score._observers['after delete']).to.be.empty();
+    });
```

Comment:
nit: extract to shared fn to end of describe block (ie. `afterEach(verifyScore)`) -- https://loopback.io/doc/en/contrib/style-guide.html